snap-core/src/main/java/org/esa/snap/core/datamodel/ProductData.java

```diff
@@ -3206,6 +3208,16 @@ public Date getAsDate() {
             return getAsCalendar().getTime();
         }
 
+        /**
+         * Returns this UTC date/time value as a LocalDateTime. The method interpretes this UTC value as a MJD 2000 date
+         * (Modified Julian Day where the  first day is the 01.01.2000).
+         *
+         * @see #getAsCalendar()
+         */
+        public LocalDateTime getAsLocalDateTime() {
```

Comment:
I would prefer moving this method to `org.esa.snap.core.util.DateTimeUtils` and adding the calendar as parameter.
This keeps the ProductData interface small and it might be useful somewhere else.

Reply:
Ok. I understand.
I will made the requested changes.